apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -102,6 +102,34 @@ export class AppRoomBridge extends RoomBridge {
 		return this.orch.getConverters()?.get('users').convertById(room.u._id);
 	}
 
+	protected async getMessages(
+		roomId: string,
+		appId: string,
```

Comment:
Please put the `appId` as the last parameter for convention

Reply:
Actually, the `options` is optional, and TS complains if we keep optional before a required param 😕